frontend/uddhar-frontend/src/coordinator/components/WeatherCard.jsx

```
@@ -16,7 +16,7 @@ const weatherData = {
     ],
 };
 
-const WeatherCard = () => {
+const WeatherCard = ({}) => {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)